build.gradle.kts

```diff
@@ -59,11 +34,19 @@ subprojects {
 	apply(plugin = "com.coditory.manifest")
 	apply(plugin = "com.github.johnrengelman.shadow")
 
-	manifest {
+	project.extensions.getByType(ManifestPluginExtension::class.java).apply {
```

Comment:
The syntax used before is cleaner and easier to understand. Why the change? (same for the `LombokExtension`.

Reply:
We are not applying plugins for the root project, so synthetic properties e.g. `manifest` is not available.

Reply:
Makes sense, thanks